src/inquirer/ask.js

```diff
@@ -12,7 +12,7 @@ const askFor = async (keyFromQuestionsMap) => {
   const activeQuestion = questionsMap[keyFromQuestionsMap];
   try {
     const answer = await prompt([activeQuestion]);
-    return { answer };
```

Comment:
99% sure i either did this or told you to do it. you're fine without it, only reason i keep the habit is because if you ever want to **add** a return value from that function, you're going to have to wrap those return values in an object, but you won't have to change the code consuming the existing return if it was already in an object. 

here that may be useful, e.g. if you wanted to return `answer` **and also** some other value you got from the question or derived from its answer. however i think something like this would be best handled in a normalizer anyway and we should avoid returning anything other than the answer inquirer gives us back.

👍